pvlib/irradiance.py

```diff
@@ -630,66 +630,74 @@ def klucher(surface_tilt, surface_azimuth, dhi, ghi, solar_zenith,
             solar_azimuth):
     r'''
     Determine diffuse irradiance from the sky on a tilted surface
-    using Klucher's 1979 model
-
-    .. math::
-
-       I_{d} = DHI \frac{1 + \cos\beta}{2} (1 + F' \sin^3(\beta/2))
-       (1 + F' \cos^2\theta\sin^3\theta_z)
-
-    where
-
-    .. math::
-
-       F' = 1 - (I_{d0} / GHI)^2
-
-    Klucher's 1979 model determines the diffuse irradiance from the sky
-    (ground reflected irradiance is not included in this algorithm) on a
-    tilted surface using the surface tilt angle, surface azimuth angle,
-    diffuse horizontal irradiance, direct normal irradiance, global
-    horizontal irradiance, extraterrestrial irradiance, sun zenith
-    angle, and sun azimuth angle.
+    using the Klucher (1979) model [1]_ [2]_.
```

Comment:
We should not put references in the first sentence of a docstring.  See #911.

Can you fix it with `haydavies` too?  Sorry I overlooked that in the review of #2191.

Reply:
> We should not put references in the first sentence of a docstring. See #911.

But they seem to be working as intended...maybe something has improved in sphinx.

Reply:
> We should not put references in the first sentence of a docstring. See #911.

Noted. I'll add that instruction into the new contributing page style guide references section (in progress)

> Can you fix it with `haydavies` too? Sorry I overlooked that in the review of #2191.
done in the last commit, is that okay in this same PR?